rtv/subreddit.py

```diff
@@ -74,6 +76,16 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(name=name)
 
+    def reddittor_profile(self):
+        if self.reddit.is_logged_in():
+            try:
+                self.content = SubredditContent.from_reddittor(
+                     self.reddit, self.username, self.loader)
```

Comment:
Specific exception

    praw.errors.OAuthScopeRequired: `get_me` requires the OAuth2 scope `identity`